### PR TITLE
Feature/recommended food card

### PR DESCRIPTION
**Pull request by minza55113151**

### CODE REVIEW

File: src/modules/menu/components/SimpleCategory.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ const SimpleCategory: React.FC<CategoryProps> = ({ category, foods }) => {
     <List
       style={{
         scrollMarginTop: "112px", // very important for anchor to work
+        overflow: "hidden",
```
Comment (JiMeow) [inline src/modules/menu/components/SimpleCategory.tsx:10]: Remove this line it make transition when click food category on topnav (look weird)

File: src/modules/menu/components/RecommendedFoodCard.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            Typography: {
+              colorText: "#ffffff",
+            },
+          },
+        }}
+      >
+        <FoodNameText>{name}</FoodNameText>
+      </ConfigProvider>
+      <PriceTag>
+        <PriceText>{price}฿</PriceText>
```
Comment (JiMeow) [inline src/modules/menu/components/RecommendedFoodCard.tsx:36]: We must talk about what unit we use ( ฿ or THB or BAHT)

File: src/modules/menu/components/RecommendedFoodCard.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  line-height: 24px;
+`;
+
+const PriceTag = styled(Tag)`
+  position: absolute;
+  right: 16px;
+  bottom: 17px;
+
+  border-radius: 12px;
+  border: 1px solid var(--neutral-5, #d9d9d9);
```
Comment (JiMeow) [inline src/modules/menu/components/RecommendedFoodCard.tsx:74]: We can use css like `var(--neutral-5, #d9d9d9);` so we change it later

File: src/modules/menu/components/RecommendedFoodCard.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  background: var(--neutral-2, #fafafa);
```
Comment (JiMeow) [inline src/modules/menu/components/RecommendedFoodCard.tsx:75]: Change css

File: src/modules/menu/components/RecommendedFoodCard.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+`;
+
+const PriceText = styled(Typography.Text)`
+  display: flex;
+  padding: 1px 8px;
+  align-items: center;
+  gap: 3px;
+
+  color: var(--neutral-9, #434343);
```
Comment (JiMeow) [inline src/modules/menu/components/RecommendedFoodCard.tsx:84]: Change css

**Review by JiMeow — CHANGES_REQUESTED** [review]
Wait golfgrab review again for more details

File: src/modules/menu/components/SimpleCategory.tsx (hunk 1 above)
Comment (minza55113151) [reply src/modules/menu/components/SimpleCategory.tsx:10]: Done

File: src/modules/menu/components/RecommendedFoodCard.tsx (hunk 3 above)
Comment (minza55113151) [reply src/modules/menu/components/RecommendedFoodCard.tsx:74]: not get

File: src/modules/menu/components/RecommendedFoodCard.tsx (hunk 2 above)
Comment (minza55113151) [reply src/modules/menu/components/RecommendedFoodCard.tsx:36]: From KATID use Baht before disgust from design team

Comment (minza55113151) [reply src/modules/menu/components/RecommendedFoodCard.tsx:36]: Done

**Review by GolfGrab — APPROVED** [review]
Approve

**Review by GolfGrab — APPROVED** [review]
Approve

**Review by GolfGrab — APPROVED** [review]

**Pull request merged**